File: Java/projet_Bike_Desktop/src/entites/Reservation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package entites;

import java.sql.Date;
import java.util.Objects;


/**
 *
 * @author asus
 */
public class Reservation {
 private int idR; 
 private int ID_U;
 private int idV;
 private Date DATE_D;
 private Date DATE_F;
 private float prix;  

    public Reservation(int ID_U, int idV, Date DATE_D, Date DATE_F, float prix) {
        this.ID_U = ID_U;
        this.idV = idV;
        this.DATE_D = DATE_D;
        this.DATE_F = DATE_F;
        this.prix = prix;
    }
    public Reservation(int ID_U, int idV, Date DATE_D, Date DATE_F, float prix,int idR) {
        this.ID_U = ID_U;
        this.idV = idV;
        this.DATE_D = DATE_D;
        this.DATE_F = DATE_F;
        this.prix = prix;
             this.idR = idR;
    }
    public void setIdR(int idR) {
        this.idR = idR;
    }

    public int getIdR() {
        return idR;
    }

    public int getID_U() {
        return ID_U;
    }

    public void setID_U(int ID_U) {
        this.ID_U = ID_U;
    }

    public int getIdV() {
        return idV;
    }

    public void setIdV(int idV) {
        this.idV = idV;
    }

    public Date getDATE_D() {
        return DATE_D;
    }

    public void setDATE_D(Date DATE_D) {
        this.DATE_D = DATE_D;
    }

    public Date getDATE_F() {
        return DATE_F;
    }

    public void setDATE_F(Date DATE_F) {
        this.DATE_F = DATE_F;
    }

    public float getPrix() {
        return prix;
    }

    public void setPrix(float prix) {
        this.prix = prix;
    }

    @Override
    public String toString() {
        return "Reservation{" + "ID_U=" + ID_U + ", idV=" + idV + ", DATE_D=" + DATE_D + ", DATE_F=" + DATE_F + ", prix=" + prix + '}';
    }

    @Override
    public int hashCode() {
        int hash = 5;
        hash = 53 * hash + this.ID_U;
        hash = 53 * hash + this.idV;
        hash = 53 * hash + Objects.hashCode(this.DATE_D);
        hash = 53 * hash + Objects.hashCode(this.DATE_F);
        hash = 53 * hash + Float.floatToIntBits(this.prix);
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Reservation other = (Reservation) obj;
        if (this.ID_U != other.ID_U) {
            return false;
        }
        if (this.idV != other.idV) {
            return false;
        }
        if (Float.floatToIntBits(this.prix) != Float.floatToIntBits(other.prix)) {
            return false;
        }
        if (!Objects.equals(this.DATE_D, other.DATE_D)) {
            return false;
        }
        if (!Objects.equals(this.DATE_F, other.DATE_F)) {
            return false;
        }
        return true;
    }
 
 

  
  
    
    
    
}

File: symfony/bike/src/ReservationBundle/Entity/Reservation.php
<?php

namespace ReservationBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Reservation
 *
 * @ORM\Table(name="reservation")
 * @ORM\Entity(repositoryClass="ReservationBundle\Repository\ReservationRepository")
 */
class Reservation
{
    /**
     * @var int
     *
     * @ORM\Column(name="idR", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date_D", type="date")
     */
    private $dateD;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date_F", type="date")
     */
    private $dateF;

    /**
     * @var float
     *
     * @ORM\Column(name="prix", type="float")
     */
    private $prix;

    /**
     * @var int
     *
     * @ORM\Column(name="idV", type="integer")
     */
    private $idV;

    /**
     * @var int
     *
     * @ORM\Column(name="ID_U", type="integer")
     */
    private $ID_U;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set dateD
     *
     * @param \DateTime $dateD
     *
     * @return Reservation
     */
    public function setDateD($dateD)
    {
        $this->dateD = $dateD;

        return $this;
    }

    /**
     * Get dateD
     *
     * @return \DateTime
     */
    public function getDateD()
    {
        return $this->dateD;
    }

    /**
     * Set dateF
     *
     * @param \DateTime $dateF
     *
     * @return Reservation
     */
    public function setDateF($dateF)
    {
        $this->dateF = $dateF;

        return $this;
    }

    /**
     * Get dateF
     *
     * @return \DateTime
     */
    public function getDateF()
    {
        return $this->dateF;
    }

    /**
     * Set prix
     *
     * @param float $prix
     *
     * @return Reservation
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Set idV
     *
     * @param string $idV
     *
     * @return Reservation
     */
    public function setIdV($idV)
    {
        $this->idV = $idV;

        return $this;
    }

    /**
     * Get idV
     *
     * @return string
     */
    public function getIdV()
    {
        return $this->idV;
    }

    /**
     * @return int
     */
    public function getIDU()
    {
        return $this->ID_U;
    }

    /**
     * @param int $ID_U
     */
    public function setIDU($ID_U)
    {
        $this->ID_U = $ID_U;
    }

}

